dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -1653,6 +1657,44 @@ protected String validateAppsLoaded(List<String> appList, int intendedTimeout, i
         }
     }
 
+    private void validateTestedFeatures(RemoteFile serverLog) throws Exception {
+        final String m = "validateTestedFeatures";
+        // Load the tested feature data, if it exists
+        File testedFeaturesFile = new File("fat-feature-deps.json");
+        if (!testedFeaturesFile.exists()) {
+            Log.info(c, m, "No tested feature data for this server.  Skipping feature validation");
+            return;
```

Comment:
Would it make sense to not return here yet, but instead also check the server logs to make sure that, indeed, no features were installed?

Reply:
I debated this one while writing but ultimately decided to just bail out.  If we hit a scenario where no features were found in static config, the bucket is probably doing something very odd, so I just did not ouput a JSON file.  When computing which FATs to run, we could take the "no JSON file" case as a sign to simply run the bucket in all cases.